tgrun/pkg/runner/vmi/embed/common.sh

```diff
@@ -102,7 +102,7 @@ function wait_for_initprimary_done()
     fi
     echo "initprimary not ready"
     i=$((i+1))
-    if [ $i -gt 20 ]; then
+    if [ $i -gt 45 ]; then
```

Comment:
why not increase to 90 minutes which is the total time of a testgrid run?

Reply:
done.